### PR TITLE
New Strategies

### PR DESCRIPTION
**Pull request by AbigailTCameron**

### CODE REVIEW

**Review by zeroecksYeti — APPROVED** [review]
LGTM

File: contracts/strategies/traderJoe/strategy-joe-avax-cly.sol [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+            avax_cly_poolId,
+            joe_avax_cly_lp,
+            _governance,
+            _strategist,
+            _controller,
+            _timelock
+        )
+    {}
+
+    function _takeFeegClyToSnob(uint256 _keep) internal {
```
[the remark this answers is not in the document]
Comment (AbigailTCameron) [reply contracts/strategies/traderJoe/strategy-joe-avax-cly.sol]: yep. We use it mostly when there isn't a direct path to the token we want. And in this case, it's a double rewards pool so we're taking a fee from a token in the pool that isn't avax, so more likely than not we'll need this function

**Review by zeroecksYeti — APPROVED** [review]
LGTM

**Review by zeroecksYeti — APPROVED** [review]
Tested, reviewed, approved.

File: hardhat.config.js [hunk 2]
```diff
@@ -47,6 +47,12 @@ module.exports = {
       },
     ],
   },
+  paths: {
```
Comment (zeroecksYeti) [inline hardhat.config.js:50]: @bigwampa added paths to eliminate redundant compilation.

**Review by zeroecksYeti — APPROVED** [review]
Tested and deployed pools, everything LGTM.

**Pull request merged**